src/TerminalFighter/MissileLauncher/MissileLauncher.cpp

```diff
@@ -7,15 +7,17 @@ MissileLauncher::MissileLauncher(Team::Team team, I_GameObjectMediator& game_obj
     : team_(team)
     , game_object_mediator_(game_object_mediator)
     , terminal_((SCREEN_WIDTH / 2) - 400, SCREEN_HEIGHT - 150, 100, 30)
+    , targeting_system_(new TargetingSystem(3, 5, "FFFF"))
```

Comment:
Last one is a color code right? Use FFF to be more standard. Usually hex color codes are either 3 letters or 6 letters. If it's 3, each letter gets repeated to form the 6. 

i.e. FEA -> FFEEAA